python/ray/tune/trainable.py

```diff
@@ -86,6 +88,8 @@ def __init__(self, config=None, logger_creator=None):
         self._restored = False
         self._setup(copy.deepcopy(self.config))
         self._local_ip = ray.services.get_node_ip_address()
+        self._track_sys_usage = config["log_sys_usage"]
```

Comment:
Since the config is for user settings, can we put this somewhere else? It can be a flag in tune.run that is true by default.